src/quo2/components/settings/privacy/option.cljs

```diff
@@ -0,0 +1,67 @@
+(ns quo2.components.settings.privacy.option
+  (:require [react-native.core :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.selectors.selectors :as selectors]
+            [quo2.components.settings.privacy.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]))
+
+(defn- bullet
+  []
+  [rn/view {:style style/bullet-container}
+   [rn/view {:style (style/bullet)}]])
+
+(defn- unordered-list
+  [container-style list-items]
+  [rn/view {:style (merge style/list-container container-style)}
+   (for [item list-items]
+     ^{:key (hash item)}
+     [rn/view {:style style/list-item}
+      [bullet]
+      [text/text {:size :paragraph-2} item]])])
+
+(defn- card-radio
+  [{:keys [accessibility-label active? on-select]
+    :or   {accessibility-label "radio button"}}]
+  [rn/touchable-without-feedback
+   {:on-press            (when on-select #(on-select))
+    :accessibility-role  :radio
+    :accessibility-label (str accessibility-label (if active? "checked" "unchecked"))}
```

Comment:
Now that I'm seeing English words in the component library, I'm starting to think it's not the best experience for screen reader's users who don't understand the language. This is not a problem of your PR @gdevincenzi, and I'm not suggesting changes, but I'm raising this concern now for other reviewers.

The Status app is pretty good with internationalization, but not the *accessible* part of it, which uses English 99% of the time.

I'm taking note of this problem for future discussion. Any thoughts @flexsurfer, @Parveshdhull?

Reply:
accessibility-label is used only for testing

Reply:
but good point cc @churik

Reply:
> accessibility-label is used only for testing

I didn't know that, thanks. It feels wrong to use that important RN property for testing because it's meant for persons with disabilities. If that's the case then I think there are better ways to add *DOM-like hints* for testing, so in return we free the `accessibility-label` property to help us build a more accessible app in the future.

Reply:
> > accessibility-label is used only for testing
> 
> I didn't know that, thanks. It feels wrong to use that important RN property for testing because it's meant for persons with disabilities. If that's the case then I think there are better ways to add _DOM-like hints_ for testing, so in return we free the `accessibility-label` property to help us build a more accessible app in the future.

i think it's not our decision, that how test frameworks work,  cc @churik @qoqobolo

Reply:
Thank you for tagging! Unfortunately, we don't have pretty much choice here: https://appium.io/docs/en/writing-running-appium/finding-elements/ 

There are 2 ways: XPath and accessibility-ids. We're using both.
If smbd knows how to add other attributes to that structure, I'm totally up for that! 
Currently, only accessibility-ids are can be used as anchors to build re-usable XPath.
![Appium 2022-12-07 16-17-11](https://user-images.githubusercontent.com/4557972/206217727-ec640655-55ed-4086-ac69-e0d5867a0293.png)